packages/subgraph/scripts/generate-introspection-json.sh

```diff
@@ -34,7 +34,7 @@ graph create $SUBGRAPH_NAME-$NETWORK_NAME --node http://localhost:8020
 # deploy subgraph
 LOCAL=true ./scripts/deploy-subgraph.sh
 # make introspection json
-FILENAME="docs/schema-introspection.json"
+FILENAME="tests/utils/schema-introspection.json"
```

Comment:
Why this is located in the `tests/utils` folder? This seems wrong

Reply:
this introspection query is going to be used for testing the query examples

Reply:
but it is also used for non-test related action ? if yes it should not be in tests/

Reply:
Not for the moment, and I can't see another use for us in the future.
Introspection queries are used to retrieve information about the GraphQL schema itself. Commonly used for auto-generating documentation, building dynamic queries, and performing schema validation.
We already have a different one for docs, and this one will only serve for schema validation (testing) purposes.

Reply:
Then the script `generate-introspection-json.sh` perhaps need to be moved to a scripts/tests/
WDYT?

Reply:
makes sense

Reply:
@Rekard0 fixed in https://github.com/aragon/osx/pull/578/commits/ef3e3c864bed97ca2fd77be33e9d376b50324cc7